packages/frontend/editor-ui/src/features/dataStore/module.descriptor.ts

```diff
@@ -40,7 +40,7 @@ export const DataStoreModule: FrontendModuleDescription = {
 		},
 		{
 			name: PROJECT_DATA_STORES,
-			path: 'datatables',
+			path: 'datatables/:new(new)?',
```

Comment:
is the bath being a relative(?) one with no leading / correct here (unlike on DATA_STORE_VIEW for instance) ?

Reply:
TIL, nice.